client/tsconfig.json

```
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "sourceMap": true,
+    "lib": ["ES2020", "DOM"],
+    "jsx": "react",
+    "module": "esnext",
+    "moduleResolution": "Node",
+    "target": "ES5",
+    "strict": true,
+    "resolveJsonModule": true,
+    "typeRoots": ["./node_modules/@types"],
+    "baseUrl": "./src",
+    "paths": {
+      "@/*": ["./*"]
+    }
+  },
+  "include": ["./src/**/*", "images.d.ts"],
+  "exclude": ["node_modules"]
+}
```

```suggestion
{
  "compilerOptions": {
    "esModuleInterop": true,
    "noEmit": true,
    "skipLibCheck": true,
    "isolatedModules": true,
    "lib": ["ES2020", "DOM"],
    "jsx": "react",
    "moduleResolution": "Node",
    "target": "ES5",
    "strict": true,
    "resolveJsonModule": true,
    "baseUrl": "./src",
    "paths": {
      "@/*": ["./*"]
    }
  },
  "include": ["./src/**/*", "images.d.ts"],
}
```

`babel`을 사용해서 typescript를 transpiling하기 때문에 세팅을 좀더 추가하고 빼야합니다.
`noEmit: ture`를 지정하여 tsc를 type checking용으로 변경하세요.
`skipLibCheck: true`를 지정하여 `node_modules/@types`의 오류를 피하세요.
`isolatedModules: true`를 지정하여 typescript를 module로만 사용할 수 있게 만드세요.
`sourceMap`은 필요하지 않습니다. webpack이 해줄겁니다.
`typeRoots`의 기본값과 동일하게 설정했으므로 필요하지 않습니다.
`exclude`의 기본값과 동일하므로 필요하지 않습니다.